config/manifests/bases/camel-k.clusterserviceversion.yaml

```diff
@@ -26,7 +26,7 @@ metadata:
     createdAt: 2022-09-21T12:01:32Z
     description: Red Hat Integration - Camel K is a lightweight integration platform,
       born on Kubernetes, with serverless superpowers.
-    olm.skipRange: '>=1.6.7 <1.8.0'
+    olm.skipRange: '>=1.8.0 <1.10.0'
```

Comment:
Is this skipRange still required? I think we introduced at the time of 1.6.9 failure.